node/src/proxy_client/stream_handler_pool.rs

```diff
@@ -347,6 +350,15 @@ impl StreamHandlerPoolReal {
             }
             Ok(lookup_ip) => lookup_ip.iter().collect(),
         };
+
+        if ip_addrs[0] == IpAddr::V4(Ipv4Addr::new(0, 0, 0, 0)) {
+            error!(
```

Comment:
First, I don't think this is an error for the exit Node. Even `warn!` seems a little dramatic. How about just `info!`?

Second, not everybody knows what "wildcard IP address" means. Can you modify the message to talk about IP blocking?